Sky Atlas/Sky Atlas.html

```diff
@@ -21209,7 +21264,7 @@ <h1>Agent Scope Database</h1>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td><p>The SPK Company Ltd account will transfer all of the SPK tokens held by it, other than the tokens specified in Transfer Of Tokens For Token Launch, to the Spark Foundation. The SPK Company Ltd account is specified in <dfn>Minting Of Tokens To SPK Company Ltd</dfn>.</p><p>The address of the Spark Foundation on the Ethereum Mainnet will be specified in a future iteration of the Spark Artifact.</p></td>
+          <td><p>The SPK Company Ltd account will transfer all of the SPK tokens held by it, other than the tokens specified in Transfer Of Tokens For Token Launch, to the Spark Foundation. The SPK Company Ltd account is specified in <dfn>Minting Of Tokens To SPK Company Ltd</dfn>.</p><p>The address of the Spark Foundation on the Ethereum Mainnet is <code>0x92e4629a4510AF5819d7D1601464C233599fF5ec</code>.</p></td>
```

Comment:
an existing issue but we may as well fix it while we are here

```suggestion
          <td><p>The SPK Company Ltd account will transfer all of the SPK tokens held by it, other than the tokens specified in <dfn>Transfer Of Tokens For Token Launch</dfn>, to the Spark Foundation. The SPK Company Ltd account is specified in <dfn>Minting Of Tokens To SPK Company Ltd</dfn>.</p><p>The address of the Spark Foundation on the Ethereum Mainnet is <code>0x92e4629a4510AF5819d7D1601464C233599fF5ec</code>.</p></td>
```